examples/packages/WORKSPACE

```diff
@@ -23,6 +23,12 @@ npm_install(
     package_lock_json = "//:package-lock.json",
 )
 
+npm_install(
+    name = "npm_install_dup",
```

Comment:
also consider a longer name like `npm_install_duplicate_for_determinism_testing` - no one needs to type this out so it's better to add documentation in the name

Reply:
👍